### PR TITLE
Update General Ranking Criteria from Drafts

### PR DESCRIPTION
**Pull request by Okorin**
### Description

Applies all bubbled drafts to the general ranking criteria: that being timing, skinning and storyboarding

### Status

Finished

### Changelog

- changes structure to list rules and guidelines per part of rc instead of a huge chunk of guidelines at the bottom
- adds glossary
- sorts existing guidelines if they aren't mode specific
- mode specific guidelines were dropped
- adds timing draft + glossary https://osu.ppy.sh/community/forums/topics/557042
- adds storyboarding draft + glossary https://osu.ppy.sh/community/forums/topics/559005
- adds skinning draft + glossary https://osu.ppy.sh/community/forums/topics/602559
- changes everything in "" to be in `comments`
- masks technical terms the same way

### CODE REVIEW

**Comment by Okorin** [comment]
note for myself: someone said that like one guideline is basically existing in two separate sections of it - i'd prefer to first update all the sections and then see where and what can be done to duplicated things

File: wiki/Ranking_Criteria/en.md [hunk 1]
```diff
@@ -36,87 +81,113 @@ All rules are exactly that: **rules**. They are **not** guidelines and may **not
...
 -   **Eastern artists must be written in the proper 'surname firstname' format** (e.g. Japanese/Chinese/Korean). For artists which have a preferred romanisation, use this one. If you're unsure of your artist's name, then check previously ranked maps, Google, Wikipedia, or ask a BN for help.
--   **Songs with Japanese titles must use the Modified Hepburn method of romanisation in the "Romanised Title" field.** As a non-unicode field, long vowels such as "おう" and "うう" should be romanised into "ou" and "uu" to avoid macrons. ([Refer to this link for more information](https://en.wikipedia.org/wiki/Hepburn_romanization#Features_of_Hepburn_romanization)). Loan words should be expressed using romanisation from the original language.
+-   **Songs with Japanese titles must use the Modified Hepburn method of romanisation in the `Romanised Title` field.** As a non-unicode field, long vowels such as `おう` and `うう` should be romanised into `ou` and `uu` to avoid macrons. ([Refer to this link for more information](https://en.wikipedia.org/wiki/Hepburn_romanization#Features_of_Hepburn_romanization)). Loan words should be expressed using romanisation from the original language.
```
Comment (MegaApplePi) [inline wiki/Ranking_Criteria/en.md]: - remove the period before the "Refer to" link in round brackets
- change `Refer to this link for more information` to `Hepburn romanization` or similar
- also, the `#Features_of_Hepburn_romanization` of the link above doesn't exist ... anymore?

File: wiki/Ranking_Criteria/en.md [hunk 2]
```diff
@@ -36,87 +81,113 @@ All rules are exactly that: **rules**. They are **not** guidelines and may **not
...
 -   **Songs with Chinese titles must be romanised in this method: each character must be romanised into a single, capitalised, separated word.** The official romanization should be used if provided. Loan words should be expressed using romanisation from the original language. Refer to [Thread: Romanisation of Chinese](https://osu.ppy.sh/forum/t/244684) for more information.
```
Comment (MegaApplePi) [inline wiki/Ranking_Criteria/en.md]: update link to `https://osu.ppy.sh/community/forums/topics/244684`  .... unless the old site is prefered at this time
 🤷‍♂️

File: wiki/Ranking_Criteria/en.md [hunk 3]
```diff
@@ -36,87 +81,113 @@ All rules are exactly that: **rules**. They are **not** guidelines and may **not
...
 -   **The minimum draining (play) time for a map is 30 seconds** (but the recommended minimum is 45 seconds). If your map is shorter than that, then try looping the song with an audio editing program. This is so each map gives a high enough score, and also so people do not try and cut songs too short for no reason, thus stopping people from enjoying the song fully.
 -   **You must cut your `.mp3` if you're using less than 80% of it.** This doesn't count intro time, and thus only applies if more than 20% of the outro is unmapped. If you only plan to map a portion of your song, then including the full `.mp3` is a waste of file space. Many `.mp3` editing tools [like this](https://osu.ppy.sh/forum/viewtopic.php?f=56&t=34303) exist which make fading and cutting a song simple. However, if you have a legitimate reason for keeping your full `.mp3` (e.g. a storyboard after the map ends), then that is fine.
```
Comment (MegaApplePi) [inline wiki/Ranking_Criteria/en.md]: - probably should fix the link to `https://osu.ppy.sh/forum/t/34303` (old site) or `https://osu.ppy.sh/community/forums/topics/34303` (new site)
- change the link text `like this` to `like mp3DirectCut`

File: wiki/Ranking_Criteria/en.md [hunk 4]
```diff
@@ -36,87 +81,113 @@ All rules are exactly that: **rules**. They are **not** guidelines and may **not
...
+-   **Hitsounds must be in `.wav`.** Hitsounds in `.wav` format are preferred, as `.mp3` files will not loop correctly and have a short delay, between 0ms and 20ms, but `.mp3` is allowed only for a special purpose for the beatmap, like the usage of `reverse cymbal` that have delay on the start. Note that using ogg is unrankable.
 -   **Hitsounds from notes and sliders must be audible.** These provide feedback for the player, and having them silent in a rhythm game doesn't make much sense. If you don't like the default sounds, then find replacements rather than silencing notes. You can use hitsounds from the [Custom Hitsound Library](https://osu.ppy.sh/forum/viewtopic.php?f=56&t=15407) or easily find others online. Lowering the volume of a few notes to provide a dampened effect is usually fine, but complete silence is always unacceptable. The end of a spinner (or even the entire spinner) the sliding sound of a slider, and the end of a slider can be silent, but only do it if it makes sense. Finally, you cannot silence both slider ticks and slider slides together.
```
Comment (MegaApplePi) [inline wiki/Ranking_Criteria/en.md]: change the link to `https://osu.ppy.sh/forum/t/15407` or `https://osu.ppy.sh/community/forums/topics/15407`

File: wiki/Ranking_Criteria/en.md [hunk 5]
```diff
@@ -36,87 +81,113 @@ All rules are exactly that: **rules**. They are **not** guidelines and may **not
...
+-   **Skinned elements must be cropped cleanly so they do not have pixelated artifacts around them or half-cropped shadows.**
+-   **Do not make any gameplay elements larger than their recommended skin dimensions.** Elements like this may affect gameplay performance, and thus having them in a different size should be avoided. Elements that are not tagged as gameplay-relevant may use a different dimension and if no recommended size is given; this variable is left up to the skinner.
+-   **When skinning gameplay elements, complete sets of elements need to be skinned in order to avoid conflicts between user-specific and beatmap-specific skins.** A reference for this can be found [here](/wiki/Ranking_Criteria/Skin_Set_List/). When skinning an element that is marked as optional, you need to include all the required elements of the respective set, but you are free to skip other optional elements unless they are grouped with the element you are skinning.
```
Comment (MegaApplePi) [inline wiki/Ranking_Criteria/en.md]: you probably should use `Skin Set List` instead of `here` as the link text 🤷‍♂️

**Pull request merged**